### PR TITLE
8.0 afr parser tests hbto

### PR DESCRIPTION
**Pull request by hbto**
Providing Test for Parser in AFR for only the code that is really being used
Later Clean Up of Code is to be Applied

### CODE REVIEW

**Comment by hbto** [comment]
This PR is addressing this issue https://github.com/Vauxoo/odoo-afr/issues/20

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ install:
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
   - pip install beautifulsoup4
+  - pip install xlrd
```
Comment (moylop260) [inline .travis.yml:39]: FYI you could use `requirements.txt` file from root project.

File: account_financial_report/__openerp__.py [hunk 2]
```diff
@@ -39,6 +39,7 @@
         "security/security.xml",
         "security/ir.model.access.csv",
         "data/report_paperformat.xml",
+        "demo/demo.xml",
```
Comment (moylop260) [inline account_financial_report/__openerp__.py]: Following the [file-naming](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming) change: `s/demo.xml/account_demo.xml/g`

File: .travis.yml [hunk 3]
```diff
@@ -36,6 +36,7 @@ install:
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
   - pip install beautifulsoup4
```
Comment (moylop260) [inline .travis.yml:38]: I don't found where you are using `bs4`
Could we take advantage of this PR to remove it?

**Comment by moylop260** [comment]
Cool `Coverage increased (+48.6%)`!

File: .travis.yml (hunk 3 above)
Comment (hbto) [inline .travis.yml:38]: @moylop260 Doing at https://github.com/Vauxoo/odoo-afr/pull/34
where cleansing on this branch will be performed
Regards

**Comment by nhomar** [comment]
hbto.

The topic of get rid of Beatifulsoup as dependency was not solved already?

**Comment by hbto** [comment]
@nhomar 
@moylop260 is referring to .travis file dependency.

that is being taking care at https://github.com/Vauxoo/odoo-afr/pull/34/files#diff-354f30a63fb0907d4ad57269548329e3L38
in https://github.com/Vauxoo/odoo-afr/pull/34

**Pull request merged**